sdk/manifest.go

```diff
@@ -58,9 +58,30 @@ type EncryptionInformation struct {
 	IntegrityInformation `json:"integrityInformation"`
 }
 
+type Statement struct {
+	Format StatementFormat `json:"format,omitempty"`
+	Schema Schema          `json:"schema,omitempty"`
```

Comment:
I think these two should be strings; both of theses could be custom per the deployment domain and the enumerated types defined are confusing b/c they both restrict us and don't actually contain enumerated types we know are needed in current day customer deployments